docs/en/platform-engineers/openapi-v3-json-schema.md

```diff
@@ -0,0 +1,136 @@
+
+For example, we can generate OpenAPI v3 JSON Schema from `parameter` of Workload Type [webservice](https://kubevela.io/#/en/developers/references/workload-types/webservice).
+
+```json
+{
+	"properties": {
+		"cmd": {
+			"description": "Commands to run in the container",
+			"items": {
+				"type": "string"
+			},
+			"title": "cmd",
+			"type": "array"
+		},
+		"cpu": {
+			"description": "Number of CPU units for the service, like `0.5` (0.5 CPU core), `1` (1 CPU core)",
+			"title": "cpu",
+			"type": "string"
+		},
+		"env": {
+			"description": "Define arguments by using environment variables",
+			"items": {
+				"properties": {
+					"name": {
+						"description": "Environment variable name",
+						"title": "name",
+						"type": "string"
+					},
+					"value": {
+						"description": "The value of the environment variable",
+						"title": "value",
+						"type": "string"
+					},
+					"valueFrom": {
+						"description": "Specifies a source the value of this var should come from",
+						"properties": {
+							"secretKeyRef": {
+								"description": "Selects a key of a secret in the pod's namespace",
+								"properties": {
+									"key": {
+										"description": "The key of the secret to select from. Must be a valid secret key",
+										"title": "key",
+										"type": "string"
+									},
+									"name": {
+										"description": "The name of the secret in the pod's namespace to select from",
+										"title": "name",
+										"type": "string"
+									}
+								},
+								"required": ["name", "key"],
+								"title": "secretKeyRef",
+								"type": "object"
+							}
+						},
+						"required": ["secretKeyRef"],
+						"title": "valueFrom",
+						"type": "object"
+					}
+				},
+				"required": ["name"],
+				"type": "object"
+			},
+			"title": "env",
+			"type": "array"
+		},
+		"image": {
+			"description": "Which image would you like to use for your service",
+			"title": "image",
+			"type": "string"
+		},
+		"port": {
+			"default": 80,
+			"description": "Which port do you want customer traffic sent to",
+			"title": "port",
+			"type": "integer"
+		}
+	},
+	"required": ["image", "port"],
+	"type": "object"
+}
+```
+
+Rendering the schema by [form-render](https://github.com/alibaba/form-render) or [React JSON Schema form](https://github.com/rjsf-team/react-jsonschema-form), a web form will show as below.
+
+![](../../resources/json-schema-render-example.jpg)
+
+## How to get the JSON Schema of a definition
+
+When a platform builder applies a WorkloadDefinition or TraitDefinition, a ConfigMap will be created in a namespace same
+as the definition (currently it's `vela-system` as definition are not namespaced) , and the name is in the format
+`schema-$definitionName`. The key of ConfigMap data is `openapi-v3-json-schema`.
+
+For example, we can use the following command to get the JSON Schema of `webservice`.
+
+```shell
+$ kubectl get cm schema-webservice -n vela-system -o yaml
```

Comment:
please add a label on the configmap, so we can list all configmaps generated by KubeVela.

```
$ kubectl get cm -n vela-system -l definition.oam.dev=schema
NAME               DATA   AGE
schema-webservice   1       2d
schema-worker         1       1d
```

Reply:
Good point!